skaffold.yaml

```diff
@@ -29,7 +29,7 @@ deploy:
         createNamespace: true
         remoteChart: rabbitmq
         repo: https://charts.bitnami.com/bitnami
-        version: 10.3.9
+        version: 12.2.5
```

Comment:
There's a newer version `12.4.1` now, should we use it?

Reply:
Most certainly! I guess I forget to update my helm repo before searching, thanks :pray: